[BUG] Exception in CLI when using TS example

**Describe the bug**

If `Configuration/TypoScript/Examples/Suggest/setup.typoscript` is included, cli mode scheduler jobs produce an exception:

> Tue, 12 Dec 2023 12:26:37 +0100 [ERROR] request="9f57bf1ef9f21" component="TYPO3.CMS.Scheduler.Scheduler": Task failed to execute successfully. Class: DirectMailTeam\DirectMail\Scheduler\DirectmailScheduler, UID: 3- RuntimeException: Unable to call method "getQueryParams" of non-object "request"., in file /home/prod/vendor/symfony/expression-language/Node/GetAttrNode.php:110 - {"class":"DirectMailTeam\\DirectMail\\Scheduler\\DirectmailScheduler","uid":3,"exception":"RuntimeException: Unable to call method \"getQueryParams\" of non-object \"request\". in /home/prod/vendor/symfony/expression-language/Node/GetAttrNode.php:110\nStack trace:\n#0 /home/prod/vendor/symfony/expression-language/Node/FunctionNode.php(47): Symfony\\Component\\ExpressionLanguage\\Node\\GetAttrNode->evaluate(Array, Array)\n#1 /home/prod/vendor/symfony/expression-language/Node/BinaryNode.php(98): Symfony\\Component\\ExpressionLanguage\\Node\\FunctionNode->evaluate(Array, Array)\n#2 /home/prod/vendor/symfony/expression-language/ExpressionLanguage.php(59): Symfony\\Component\\ExpressionLanguage\\Node\\BinaryNode->evaluate(Array, Array)\n#3 /home/prod/vendor/typo3/cms-core/Classes/ExpressionLanguage/Resolver.php(73): Symfony\\Component\\ExpressionLanguage\\ExpressionLanguage->evaluate('traverse(reques...', Array)\n#4 /home/prod/vendor/typo3/cms-core/Classes/TypoScript/IncludeTree/Visitor/IncludeTreeConditionMatcherVisitor.php(162): TYPO3\\CMS\\Core\\ExpressionLanguage\\Resolver->evaluate('traverse(reques...')\n

While this may see to be a direct_mail issue, in fact, it is due to using `request` object in a condition while there is not request:

```
[traverse(request.getQueryParams(), 'tx_solr/callback') == '']
```

Instead it should be:

```
[request && traverse(request.getQueryParams(), 'tx_solr/callback') == '']
```

PR is to be provided.

**To Reproduce**

Not sure if it can be done without direct_mail but with direct_mail it is reproducible if that TS is included and scheduler task is set up to send mails.

**Used versions (please complete the following information):**
 - TYPO3 Version: 12.4.8
 - EXT:solr Version: 12.0.1
 - EXT:direct_mail: 12.x-dev from https://github.com/dmitryd/direct_mail.git

--- a/Configuration/TypoScript/Examples/Suggest/setup.typoscript
+++ b/Configuration/TypoScript/Examples/Suggest/setup.typoscript
@@ -23,7 +23,7 @@ tx_solr_suggest {
     }
 }
 
-[traverse(request.getQueryParams(), 'tx_solr/callback') == '']
+[request && traverse(request.getQueryParams(), 'tx_solr/callback') == '']
     tx_solr_suggest.config.additionalHeaders.10.header = Content-type: application/json
 [global]
 

